mypy/checkexpr.py

```diff
@@ -1061,15 +1061,51 @@ def check_op(self, method: str, base_type: Type, arg: Node,
             # method of the other operand instead.
             rmethod = self.get_reverse_op_method(method)
             arg_type = self.accept(arg)
+            base_arg_node = TempNode(base_type)
+            user_error = None  # type: Tuple[Tuple[Type, Type], MessageBuilder]
```

Comment:
Also, the name of the variable is a little confusing. Maybe the name could be more informative? Alternatively, add a comment.